src/anm/analytics.js

```diff
@@ -16,27 +13,15 @@ var Analytics = function () {
             var data = JSON.stringify(self.queue);
             self.queue = [];
 
-            if (supportSendBeacon) {
+            if (navigator.sendBeacon) {
                 navigator.sendBeacon(animatronUrl, data);
             } else {
-                sendViaGif(data);
+                engine.ajax(animatronUrl, null, null, 'POST', null, data, false);
             }
         }
-        if (!supportSendBeacon) {
-            setTimeout(event, timeout);
-        }
     };
 
-    var sendViaGif = function (data) {
-        if (!beacon) {
-            beacon = engine.createStatImg();
```

Comment:
Ok, good then. Let's test it in the action finally :).

> since the image way won't work with `unload` event.

I wasn't aware of that. Anyway, my point was is in unnecessary timeout calls, so no, not useless in the end ;).